discord/commands/context.py

```diff
@@ -142,8 +148,8 @@ def guild_locale(self) -> Optional[str]:
         return self.interaction.guild_locale
 
     @cached_property
-    def me(self) -> Union[Member, User]:
-        return self.guild.me if self.guild is not None else self.bot.user
+    def me(self) -> Union[Member, ClientUser, None]:
```

Comment:
Wouldn't it be better to use Optional here?

Reply:
Optional is just an alias for Union[T, None] and unions get flattened anyway. I can change it if you want.

Reply:
I said it for consistency with the other attributes, so yeah